docker/main/ngen/Dockerfile

```diff
@@ -664,6 +664,10 @@ RUN if [ "${NGEN_WITH_PYTHON}" == "ON" ]; then \
     fi
 USER ${USER}
 
+ENV VIRTUAL_ENV=/dmod/venv
+RUN python3 -m venv $VIRTUAL_ENV && pip3 install numpy
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
 RUN cd ${BOOST_ROOT} \
```

Comment:
It seems like activating the virtual environment here before doing the build and then change 856 to `ENTRYPOINT ["source $VIRTUAL_ENV/bin/activate", "&&", "entrypoint.sh"]`?

Reply:
Can you clarify what you are asking here?